src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -44,6 +44,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.getWriter().println(gson.toJson(newTrip.tripID()));
         } 
         else {
+            System.out.println(request.getParameter(START_DATE_PARAM));
+            System.out.println("Input information error: "+ error);
```

Comment:
Do you think we should remove the print statements or do you think they are good things too log?

Reply:
i think we can keep it for now? after mvp, we can do a cleanup